packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -789,31 +795,53 @@ class PaddleSliderValueIndicatorShape extends SliderComponentShape {
     );
     final double leftNeckArcAngle = _ninetyDegrees - leftTheta;
     final double rightNeckArcAngle = math.pi + _ninetyDegrees - rightTheta;
+    // How much distance there is between the end of the bottom neck arc and the
+    // beginning of the top neck arc. We use this to shrink/expand it based on
+    // the scale factor of the value indicator.
+    final double neckStretchBaseline = bottomLobeEnd.dy - math.max(neckLeftCenter.dy, neckRightCenter.dy);
+    final double t = math.pow(inverseTextScale, 3.0);
+    final double stretch = (neckStretchBaseline * t).clamp(0.0, 10.0 * neckStretchBaseline);
+    final Offset neckStretch = new Offset(0.0, neckStretchBaseline - stretch);
+
+    if (_debuggingLabelLocation) {
```

Comment:
I don't think this code should be part of the released product. Typically we put debugging helpers in an assert like:

```dart
assert(() {
  if (_debuggingLabelLocation) {
    ...
  }
}());
```

Since there's just one occurrence of this (rather helpful looking) debugging feature you could do something like

```dart
// Remove "false &&" if you want ...
assert(false && () {
  ...
}());

```

Reply:
Sounds good.  I did it this way, because I assumed that if a static const bool was false, that the compiler would remove it.

Reply:
Would this be ok?

```dart
assert(_debuggingLabelLocation && () {
  ...
}());
```

Reply:
Including the constant doesn't really seem necessary, but if you think it makes it clearer, sure.